README.md

```diff
@@ -19,9 +19,18 @@ License
 Merit Core is released under the terms of the MIT license. See [COPYING](COPYING) for more
 information or see https://opensource.org/licenses/MIT.
 
+Get Started
+-------------------
+So you want to get involved with Merit?
```

Comment:
Talk about current state of merit core and that you need to clone the repository, build the software, and run it. Maybe provide links to building and running.